src/albums-webui.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Page struct for the web server
+type Page struct {
+	Title string
+	Body  []byte
+}
+
+// AlbumWebUI web interface for the UI
+type AlbumWebUI struct {
+	album  *Albums
+	logger *logrus.Entry
+}
+
+// NewAlbumWebUI , return Albums struct
+func NewAlbumWebUI(album *Albums, logger *logrus.Entry) *AlbumWebUI {
+	return &AlbumWebUI{
+		album:  album,
+		logger: logger,
+	}
+}
+
+// RunWebSrv , provide the web interface
+func (a *AlbumWebUI) RunWebSrv() {
+	http.HandleFunc("/", a.viewHandler)
+	a.logger.Fatal(http.ListenAndServe(":8080", nil))
+}
+
+func (a *AlbumWebUI) loadPage(url string) (*Page, error) {
+
+	if url == "/" {
```

Comment:
here it will be a case to manage multiple URLs and we need to use some templating.
Honestly, it's the first time I create a web interface in GoLang, so baby step, to confirm everything works as expected.